samples/ApiGatewayHttpApiV2Tracing/Voxel.MiddyNet.ApiGatewayHttpApiV2TracingSample/serverless.yml

```diff
@@ -0,0 +1,24 @@
+service: voxel-middynet-agwv2tracingsample
+
+provider:
+  name: aws
+  runtime: dotnetcore3.1
+  region: ${opt:region, self:custom.defaultRegion}
+  stage: ${opt:stage, self:custom.defaultStage}
+
+custom:
+  defaultRegion: eu-west-1
+  defaultStage: dev${env:SLSUSER, ""}
+
+package:
+  individually: true
+
+functions:
+  apiGatewayTracingSample:
+    handler: Voxel.MiddyNet.ApiGatewayHttpApiV2TracingSample::Voxel.MiddyNet.ApiGatewayHttpApiV2TracingSample.ApiGatewayHttpApiV2Tracing::Handler
+    package:
+      artifact: bin/Release/netcoreapp3.1/Voxel.MiddyNet.ApiGatewayHttpApiV2TracingSample.zip
+    events:
+      - http:
```

Comment:
Have we tried this? The httpApi event is httpApi, not http.

Reply:
I'll try it, but I think @josep-bocanegra is right. It's the same construct we used for ApiGatewayTracingSample.

Reply:
I mean, if we´re doing this for the "new" HTTP Api, the event should be httpApi, as explained here: https://www.serverless.com/framework/docs/providers/aws/events/http-api/

Reply:
Fixed!